test/sp.config.configLoader_test.js

```diff
@@ -52,9 +51,16 @@ describe('Configuration loader', function () {
     }
   });
 
-  it('should load without configuration', function(done) {
+  it('should load configuration with only api key', function(done) {
```

Comment:
I fully agree. The thing is that this test did not fully work before, since it actually got configuration from env. But I'm instead changing the test. Will have an update to this in a few minutes.